### PR TITLE
Add/user col user connected

### PR DESCRIPTION
**Pull request by dereksmart**
This adds a blank column to the user admin table that will show a green Jetpack icon is the user is linked to a WordPress.com account.  

jetpack-user-col

This PR is intended to provide a very simple functionality.  Show a logo if user is linked.  Tooltip on hover reads an explanation, currently: `This user is linked and ready to fly with Jetpack.`.  Let's look at adding more functionality to this once the base 4.0 is merged in and we know how it'll tie in better with user connection stuff. 

addresses #2601

### CODE REVIEW

File: class.jetpack.php [hunk 1]
```diff
@@ -7042,4 +7047,53 @@ public static function jetpack_site_icon_available_in_core() {
...
+		if ( 'users' == $current_screen->base ) { ?>
+			<style>
+				.fixed .column-user_jetpack {
+					width: 21px;
+				}
+				.jp-emblem-user-admin path {
+					fill: #8cc258;
+				}
+			</style>
+		<? }
```
Comment (kraftbj) [inline class.jetpack.php]: Can't use shorttags.

**Comment by jeherve** [comment]
I made a couple more changes, including a link to "My Jetpack".

Would it be worth adding an emblem for the current user as well, but keep it grayed out if you're not connected to WordPress.com, with the same link to "My Jetpack"? It might be a good incentive for folks to link their own account as well.

**Comment by dereksmart** [comment]
@jeherve good suggestions!  How about we keep this branch as "base" functionality for now until 4.0 is further developed and we can implement more fanciness when we know how it will tie in better?

**Comment by jeherve** [comment]
This looks good to merge. 👍

**Pull request merged**